runtime/src/bank.rs

```diff
@@ -8731,12 +8731,13 @@ pub(crate) mod tests {
             .accounts_index
             .purge_roots(&zero_lamport_pubkey);
 
-        let some_slot = 1000;
+        let some_slot = 32; // 1 epoch
```

Comment:
Would it be possible to use fields within `genesis_config.epoch_schedule` to compute this value?

Reply:
used a constant